docs/user-guide/aroma/finalizing-setup.md

```diff
@@ -2,31 +2,41 @@
 
 ## Finalizing Setup
 
-Now that Aroma is on your SD Card, we need to set up some final things.
+Now that PayloadLoader, Environment Loader and Aroma are installed, we are going to finalize the setup.
 
-### Booting into Aroma
+We are going to make the Aroma environment start automatically when your console autoboots the Health and Safety Information app (or when manually launching it if you chose not to autoboot it) and select Wii U Menu as default as well as getting additional homebrew apps. Please note that the following steps assumes you are autobooting the Health and Safety Information app.
 
-1. Take the SD Card out of your computer and plug it into your Wii U console.
-1. Start the EnvironmentLoader.
-    - If you're autobooting into Tiramisu, this can be done by powering on your console and holding `X`.
-1. Highlight the entry called `aroma` using the D-Pad.  
-    - To make the console autoboot Aroma press `Y`.
-1. Launch Aroma by pressing `A`.
-    - If you get a red warning screen telling you that updates aren't blocked properly, you might want to block updates by following [this guide](../block-updates).
-1. On the Boot Selector, the `Wii U Menu` should already be selected, press Y to set this to your default autobooting option, then press A to launch into the Wii U Menu.
-    - To open the Boot Selector in the future, you have hold START (+) while launching Aroma.
+### Setting up PayloadLoader, Environment Loader and Aroma
+
+1. Turn on your Wii U.
+    - The Environment Loader should show up.
```

Comment:
Note that this will not be the case if the user isn't autobooting the PayloadLoader or came from an existing autobooting Tiramisu setup.

Reply:
Then I can change it to something like:
```
1. Launch the EnvironmentLoader.
    - If you are coldbooting the PayloadLoader, simply turn on your Wii U.
    - If you skipped the coldbooting steps, launch the Health and Safety Information app.
```